### DISCUSSION TITLE
MLOps best practices

### DISCUSSION THREAD

**Discussion by kannkyo** [github/india#95] [Satellite India 2021 Q&A] [answered]
Where can I learn about MLOps best practices?
Please let me know the material for beginners.

**Comment by DavideC03** [comment]
I think that a good basic material could be [https://ml-ops.org](https://ml-ops.org/).

**Reply by kannkyo** [reply]
Thank you for your introduction. I will try to read https://ml-ops.org/ first.

**Comment by ShrutiC-git** [comment, marked as answer]
Hi! Great question: there are some really amazing sites you can go through to read more about the MLOps ecosystem. Here's one: https://ml-ops.org/ (this is for some basic reading). Also Introducing MLOps by OReilly is a pretty good book if you want to get more in depth about MLOps!

**Reply by kannkyo** [reply]
Thanks for the interesting presentation, I've never heard of dvc before. (I installed it on my PC just now...). 
I will try to read https://ml-ops.org/ first. After that, I would like to refer to your repository https://github.com/ShrutiC-git/MLOps .

**Reply by carrycooldude** [reply]
> Hi! Great question: there are some really amazing sites you can go through to read more about the MLOps ecosystem. Here's one: https://ml-ops.org/ (this is for some basic reading). Also Introducing MLOps by OReilly is a pretty good book if you want to get more in depth about MLOps!

Thanks, Shruti It's a great session by you and I got some really Insights about MLOPs
https://twitter.com/carrycooldude/status/1375707542806917125?s=20
